packages/bpk-docs/src/components/DocsPageBuilder/PlatformSwitchingContent.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import { type Node } from 'react';
+
+const PLATFORMS = {
+  android: 'android',
+  ios: 'ios',
+  web: 'web',
+};
+
+type Props = {
+  platform: string,
+  content: {
+    android: Node,
+    ios: Node,
+    web: Node,
+  },
+};
+
+const PlatformSwitchingContent = (props: Props) => {
+  const { platform, content } = props;
```

Comment:
can we replace this with `return content[platform] || content.web`?

Reply:
If we were playing code golf, you'd have just scored a hole-in-one. Nice one 😄

Reply:
Done.